### ISSUE TITLE
add installation instructions for gdal with pixi.sh

### ISSUE THREAD

**Issue by gregorywaynepower** [OSGeo/gdal#11551]
### Feature description

I would like installation instructions via [pixi.sh](https://pixi.sh/latest/) to be included in the documentation. It's a tool made by prefix.dev that functions [similarly to conda/mamba ](https://pixi.sh/latest/switching_from/conda/). It allows users to install gdal within a project, or as a globally available tool.

**Comment by rouault** [comment]
> I would like installation instructions via [pixi.sh](https://pixi.sh/latest/) to be included in the documentation

by whom? If you're volunteering, why not. As far as I'm concerned, I had never heard about it before

**Comment by gregorywaynepower** [comment]
Yeah, I'm working on a PR for this request. Sorry if the issue sounded a tad demanding. 😅

**Comment by rouault** [comment]
@gregorywaynepower are you still planning issuing a PR / should we keep this ticket opened ?

**Comment by gregorywaynepower** [comment]
Sorry for the delay--I do plan on issuing a PR.
